TCAT/Model/BusLocation.swift

```diff
@@ -42,15 +42,45 @@ class BusLocation: NSObject, NSCoding {
     
```

Comment:
Almost. Just n2 remove the default values from the variables that are in the initializer